app/src/main/java/org/mozilla/focus/autocomplete/AutocompletePreference.kt

```diff
@@ -25,8 +25,7 @@ class AutocompletePreference(context: Context?, attrs: AttributeSet?) : Preferen
 
         summaryView = view?.findViewById(android.R.id.summary)
         summaryView?.setText(
-                if (settings.shouldAutocompleteFromShippedDomainList()
-                        || settings.shouldAutocompleteFromCustomDomainList())
+                if (settings.shouldAutocompleteFromShippedDomainList() || settings.shouldAutocompleteFromCustomDomainList())
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.